.github/workflows/golangci-lint.yaml

```
@@ -18,4 +18,4 @@ jobs:
       - name: golangci-lint
         uses: golangci/golangci-lint-action@3a919529898de77ec3da873e3063ca4b10e7f5cc # v3.7.0
         with:
-          version: v1.54
+          version: v1.55.1
```

```suggestion
          version: v1.55
```

then it will pick that patch releases automatically and we dont need to bump everytime

---

I'm cool with that too! As long as the local results match CI's results